### DISCUSSION TITLE
Internal server error on pretix small-scale deployment via docker

### DISCUSSION THREAD

**Discussion by akolaarthurali** [pretix/pretix#4008] [Q&A] [answered]
i deployed pretix just two days ago via a small-scale deployment via docker. I got this error at the very end of the installation.I need your help on how to resolve it. I have tried my very best, but I felt i need expert help. How can I really go about this. I was testing before I make a full-scale deployment ![1000108569.png](https://github.com/pretix/pretix/assets/34062229/13eeb300-6b47-41a5-994b-185eaa391cc6)

![1000108570.png](https://github.com/pretix/pretix/assets/34062229/a6154917-418a-48fd-84bd-e0d79593dc83)

![1000108568.png](https://github.com/pretix/pretix/assets/34062229/9aa36d68-6153-489b-98ce-d5e1072a47bb)

**Comment by raphaelm** [comment, marked as answer]
pretix is unable to reach the PostgreSQL container at 172.17.0.1. Is PostgreSQL running and listening on that IP?

**Reply by akolaarthurali** [reply]
Thank you, it worked well. However it was after many hours of diligently learning how to access files and network interfaces in docker. I could have been lucky at the end.
